docs/api.md

```diff
@@ -163,38 +160,38 @@ curl -v "https://api2.publish-teacher-training-courses.service.gov.uk/api/v1/201
-science              | Integer                     | 1, 2, 3, 9, null             |
-recruitment_cycle    | Text                        |                              | 4-character year
-campus_statuses      | An array of campus statuses |                              | See the campus status entity documentation below
-subjects             | An array of subjects        |                              | See the subject entity documentation below
-provider             | Provider                    | A provider entity            | See the provider entity documentation below
-accrediting_provider | Provider                    | null or a provider entity    | See the provider entity documentation below
-age_range            | Text                        | "P", "S", "M", "O"           | Age of students targeted by this course.
+| Parameter            | Data type                   | Possible values              | Description                                      |
+| -------------------- | --------------------------- | ---------------------------- | ------------------------------------------------ |
+| course_code          | Text                        | 4-character strings          | 4-character course code                          |
+| start_month          | ISO 8601 date/time string   |                              | The month and year when the course starts        |
+| start_month_string   | Text                        | January, February, etc       | The month when the course starts as a string     |
+| name                 | Text                        |                              | Course title                                     |
+| copy_form_required   | Text                        | 'Y' or 'N'                   |
+| profpost_flag        | Text                        | "", "PF", "PG", "BO"         | Maximum of 2-characters                          |
+| program_type         | Text                        | "SC", "SS", "TA", "SD", "HE" | Maximum of 2-characters                          |
+| modular              | Text                        | "", "M"                      | Maximum of 1-character                           |
+| english              | Integer                     | 1, 2, 3, 9                   |
+| maths                | Integer                     | 1, 2, 3, 9                   |
+| science              | Integer                     | 1, 2, 3, 9, null             |
+| recruitment_cycle    | Text                        |                              | 4-character year                                 |
+| campus_statuses      | An array of campus statuses |                              | See the campus status entity documentation below |
+| subjects             | An array of subjects        |                              | See the subject entity documentation below       |
+| provider             | Provider                    | A provider entity            | See the provider entity documentation below      |
```

Comment:
Should this be changed to reflect that different data will be returned?

Reply:
I started doing that @misaka but I think it'll make things more complicated so Ive just edited the description for these slightly i.e. `See the provider entity documentation below (without address data)`